pkg/model/resource_usage_parser.go

```diff
@@ -5,39 +5,59 @@ import (
 	"strings"
 
 	"github.com/BTBurke/k8sresource"
-	"github.com/c2h5oh/datasize"
+	"github.com/dustin/go-humanize"
 )
 
-func ParseResourceUsageConfig(usage ResourceUsageConfig) ResourceUsageData {
-	return ResourceUsageData{
-		CPU:    ConvertCPUString(usage.CPU),
-		Memory: ConvertBytesString(usage.Memory),
-		Disk:   ConvertBytesString(usage.Disk),
-		GPU:    ConvertGPUString(usage.GPU),
+func ParseResourceUsageConfig(usage ResourceUsageConfig) (ResourceUsageData, error) {
```

Comment:
Rather than ignore the error this method now exposes it to the caller.

Reply:
Please check if this resolves #1758.

Reply:
Now we get an error:
```
frrist@oak ~/W/s/g/b/bacalhau (frrist/cmd/config)> cat ~/.bacalhau/config.yaml
node:
    compute:
        capacity:
            defaultjobresourcelimits:
                cpu: 7500m
                disk: 564 GB
                gpu: "4"
                memory: 52 GB
            jobresourcelimits:
                cpu: 7500m
                disk: 564 GB
                gpu: "4"
                memory: 52 GB
            queueresourcelimits:
                cpu: 15000m
                disk: 1.1 TB
                gpu: "4"
                memory: 103 GB
            totalresourcelimits:
                cpu: 7500m
                disk: 564 GB
                gpu: "4"
                memory: 52 GB
frrist@oak ~/W/s/g/b/bacalhau (frrist/cmd/config)> ./bin/darwin_arm64/bacalhau serve
14:43:08.663 | INF pkg/compute/capacity/system/provider.go:106 > cannot inspect system GPUs: nvidia-smi not installed. GPUs will not be used.
validating compute config: 1 error occurred:
        * total resource limits {CPU:7.5 Memory:52000000000 Disk:564000000000 GPU:4} exceed physical resources {CPU:8 Memory:54975581388 Disk:600723724697 GPU:0}
```